drivers/base/bootvid/i386/pc/vga.c

```diff
@@ -309,11 +259,12 @@ VgaScroll(IN ULONG Scroll)
     }
 }
 
-static VOID
+VOID
 NTAPI
-PreserveRow(IN ULONG CurrentTop,
-            IN ULONG TopDelta,
-            IN BOOLEAN Restore)
+VidpPreserveRow(
```

Comment:
```suggestion
PreserveRow(
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.